ios/engine/KMEI/KeymanEngine/resources/Keyman.bundle/Contents/Resources/keyboard.html

```diff
@@ -52,6 +52,15 @@
                     
                 //});
             }
+
+            function verifyLoaded() {
+              // During proper loads of KMW, a keyboard will be set very early on.
+              // There's a chance that the keyboard is still loading, so we double-check
+              // against the stub count.
+              if(!keyman.core.activeKeyboard && keyman.keyboardManager.keyboardStubs.length == 0) {
```

Comment:
The second part of this conditional aims to reduce "false negatives", preventing unnecessary resets.

Reply:
Will this be an issue for 3rd party Keyman Engine consumers that may not have a default keyboard loaded?

Reply:
Hmm.  So, basically an iOS equivalent to https://github.com/keymanapp/keyman/issues/3922?

If no keyboard is installed, I _believe_ that `sil_eurolatin` will be automatically loaded in its place:

https://github.com/keymanapp/keyman/blob/0c015fe5c96000aeea2e9e995e7cb79e31e77134/ios/engine/KMEI/KeymanEngine/Classes/KeymanWebViewController.swift#L946-L961

(Note the final block.)